site/src/pages/tutorials/tutorial-blog-rest-intro.mdx

```diff
@@ -0,0 +1,101 @@
+---
+type: tutorial
+tags:
+  - rest
+  - request
+  - request-converter
+  - exceptions
+  - exception-handling
+  - blocking
+level: basic
+---
+
+# REST tutorial introduction
+
+In this tutorial, you will learn to build a RESTful service with Armeria.
+Using Armeria's annotations you can build RESTful services on the go.
+Armeria features you will use in this tutorial are:
+
+- [Service annotations](/docs/server-annotated-service)
+- [Request converter](/docs/server-annotated-service#injecting-value-of-parameters-and-http-headers-into-a-java-object)
+- Response converter
+- Exception handler
+- Blocking
+
+The [sample service provided](#) is a minimal blog service with which you can create, read, update and delete blog posts.
+To keep our focus on Armeria, this tutorial and the sample service does not contain database operations.
+Instead, memory-based data operation is used.
+
+[Try building a service](#run-blog-service) by following the tutorial or have a go at [running the sample service](#try-building-blog-service-yourself) right away.
+
+## Assumptions
+
+This tutorial assumes that you have:
+
+- Experience in building services in Java
+- Used other Java frameworks for server-side programming
+- An understanding of RESTful APIs and how to implement them
+
+## Prerequisites
+
+To run and develop the sample service yourself, install the following requirements on your computer:
+
+- JDK 14
+- Gradle
+  - Set your Gradle to compile Java with the `-parameters` option
```

Comment:
I think JDK 8+ is enough. 😄 
The user does not have to preinstall the Gradle because the sample code provides [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) that manages the Gradle build.
For the `-parameters`, let's link to it.
https://armeria.dev/docs/setup/#configure--parameters-javac-option

Reply:
Done!